### PR TITLE
ft(S3C-1974): Add PyKMIP and KMIP tests to CI

### PR DESCRIPTION
**Pull request by tmacro**
## Note
As some conflicts happened, the patch for 8.0 and 8.1 is slightly different, review carefully

### CODE REVIEW

**Comment by tmacro** [comment]
@bert-e help

**Review by ploki — APPROVED** [review]

File: tests/functional/kmip/serverside_encryption.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    it('should create an encrypted bucket', done => {
+        _createBucket(bucketName, err => {
+            assert.equal(err, null, 'Expected success, ' +
+            `got error ${JSON.stringify(err)}`);
+            done();
+        });
+    });
+
+    it('should create and encrypted bucket and upload an object', done => {
```
Comment (philipyoo) [inline tests/functional/kmip/serverside_encryption.js]: minor: should create ~and~ an

**Review by philipyoo — APPROVED** [review]
lgtm

**Comment by tmacro** [comment]
@bert-e reset

**Comment by tmacro** [comment]
@bert-e approve

**Review by dora-korpar — APPROVED** [review]

**Comment by tmacro** [comment]
@bert-e poke

**Pull request merged**